specification/schema/bad-request.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Bad request response.
```

Comment:
I noticed you've added some files under `schema` but they're examples. Please move them to examples and they also have to be json because that's what we're returning.

Reply:
done